pytorch_modelsize.py

```diff
@@ -34,11 +34,16 @@ def get_output_sizes(self):
         input_ = Variable(torch.FloatTensor(*self.input_size), volatile=True)
         mods = list(self.model.modules())
         out_sizes = []
-        for i in range(1, len(mods)):
-            m = mods[i]
-            out = m(input_)
-            out_sizes.append(np.array(out.size()))
-            input_ = out
+
+        def get_output_sizes_from_module(mods):
```

Comment:
Did you intend to call this right below the definition?

Reply:
Yes, it will recursively resolve all nested ModuleList for complex model architectures.

BTW, I found the calculation from this repo is less accurate compared to ```torchsummay```. In my 3D model, this will give only a few hundred MBs while it took about 5 times GPU memories large, which is pretty similar to the outputs from ```torchsummary```. I haven't got time to look into the details on how ```torchsummary``` implemented its method. You may have a look as well if you are interested.